build_wheel.sh

```diff
@@ -366,7 +366,14 @@ if [[ -z "$EBROOTGENTOO" ]]; then
 	module --force purge
 	module load nixpkgs gcc/7.3.0
 else
-	module --force purge
```

Comment:
seems a little strange to have this in both branches now?

Reply:
```
module --force purge
module load arch/${EBVERSIONARCH-sse3}
```
should work too

Reply:
`$EBVERSIONARCH` won't be defined after `module --force purge`

Reply:
ah yes!
```
ARCH_TO_LOAD="$EBVERSIONARCH"
module --force purge
module load arch/${EBVERSIONARCH:-sse3}
```
works then?

Reply:
fixed